<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountForm</class>
 <widget class="QWidget" name="AccountForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_9">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_8">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(35, 35, 35);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIconStatus">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../res/resources.qrc">:/icons/bullet_red_16px.ico</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelStatus">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="1">
          <widget class="QProgressBar" name="progressBarEnergy">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>%v/%m (%p%)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelIconEnergy">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../res/resources.qrc">:/icons/lightning_16px.ico</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_36">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIconLife">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../res/resources.qrc">:/icons/heart_16px.ico</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QProgressBar" name="progressBarLife">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>%v/%m (%p%)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_37">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIconPods">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../res/resources.qrc">:/icons/balance_16px.ico</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QProgressBar" name="progressBarPods">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>%v/%m (%p%)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_38">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIconExperience">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../res/resources.qrc">:/icons/medal_gold_blue_16px.ico</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QProgressBar" name="progressBarExperience">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>%v (%p%)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_39">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIconKamas">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../res/resources.qrc">:/icons/coins_16px.ico</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelKamas">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_7">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(35, 35, 35);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QLabel" name="labelInfosPosition">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonDisconnection">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../res/resources.qrc">
            <normaloff>:/icons/connect_16px.ico</normaloff>:/icons/connect_16px.ico</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="labelSubscriptionDofus">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="pushButtonClose">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../res/resources.qrc">
            <normaloff>:/icons/cancel_16px.ico</normaloff>:/icons/cancel_16px.ico</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonFile">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../res/resources.qrc">
            <normaloff>:/icons/script_16px.ico</normaloff>:/icons/script_16px.ico</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonGroupSettings">
          <property name="text">
           <string>Group options</string>
          </property>
          <property name="icon">
           <iconset resource="../../../res/resources.qrc">
            <normaloff>:/icons/group_16px.ico</normaloff>:/icons/group_16px.ico</iconset>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionLoadScript">
   <property name="icon">
    <iconset resource="../../../res/resources.qrc">
     <normaloff>:/icons/script_add_16px.ico</normaloff>:/icons/script_add_16px.ico</iconset>
   </property>
   <property name="text">
    <string>Load a script</string>
   </property>
  </action>
  <action name="actionRunScript">
   <property name="icon">
    <iconset resource="../../../res/resources.qrc">
     <normaloff>:/icons/script_go_16px.ico</normaloff>:/icons/script_go_16px.ico</iconset>
   </property>
   <property name="text">
    <string>Start the script</string>
   </property>
  </action>
  <action name="actionTeleportSlavesToMaster">
   <property name="text">
    <string>Teleport the mules to the leader</string>
   </property>
   <property name="toolTip">
    <string>Teleport the mules to the leader</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonDisconnection</tabstop>
  <tabstop>pushButtonFile</tabstop>
  <tabstop>pushButtonGroupSettings</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
